docs/nodes/configuration.md

```diff
@@ -202,18 +221,34 @@ pprof-laddr = ""
 #######################################################
 [p2p]
 
+# Enable the new p2p layer.
+disable-legacy = false
```

Comment:
this is now "use-legacy" (defaults to false,)

Reply:
updated, i didn't pull master